rep-2000.rst

```diff
@@ -1065,6 +1065,126 @@ Package manager use for dependencies:
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_connextdds           | RTI Connext            | Tier 1        | Ubuntu, Windows, and macOS | All Architectures except arm64 |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_zenoh                | Eclipse Zenoh          | Tier 1        | All platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS      | Tier 2        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_gurumdds_cpp         | GurumNetworks GurumDDS | Tier 3        | Ubuntu and Windows         | All Architectures except arm32 |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.9
+
+
+Dependency Requirements:
+
++----------------+---------------------------------+-------------------------------------------------------------------+
+|                |        Required Support         |                        Recommended Support                        |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Package        |  Ubuntu Noble  |  Windows 10**  |  RHEL 9  |    macOS**    | Debian Bullseye  |   OpenEmbedded**    |
++================+================+================+==========+===============+==================+=====================+
+| CMake          |      3.27.7    |     3.22.0     |  3.20.2  |    3.14.4     |      3.25.1      | 3.22.3 / 3.16.5***  |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| EmPY           |      3.3.4     |     3.3.2      |   3.3.4  |                     3.3.4                              |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Gazebo-classic |    11.x.x*     |       N/A      |   N/A    |    11.x.x     |      11.x.x*     |         N/A         |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Gazebo         |    Harmonic*   |       N/A      |   N/A    |   Harmonic*   |     Harmonic*    |         N/A         |
```

Comment:
@azeey Can you comment on which versions of Gazebo-classic and Gazebo will be supported/recommended with Ubuntu 24.04?

Reply:
The plan was to drop Gazebo-classic and pair Jazzy with Gazebo Harmonic.

Reply:
Thanks!  In that case, @marcoag let's just drop the `Gazebo-classic` row, and then we can merge this in.

Reply:
Dropped `Gazebo-classic` row.